src/pydicom/dataelem.py

```diff
@@ -597,15 +602,11 @@ def _convert_value(self, val: Any) -> Any:
         # e.g. LUT Descriptor is 'US or SS' and VM 3, but the first and
         #   third values are always US (the third should be <= 16, so SS is OK)
         if self.tag in _LUT_DESCRIPTOR_TAGS and val:
-
-            def _skip_conversion(val: Any) -> Any:
-                return val
-
             validate_value(VR_.US, val[0], self.validation_mode)
             for value in val[1:]:
                 validate_value(self.VR, value, self.validation_mode)
 
-            return MultiValue(_skip_conversion, val)
+            return MultiValue(_pass_through, val)
```

Comment:
Interesting. I never thought about pickling problems when writing local functions, but obviously you have to...